cypress/support/step-definitions/button-steps.js

```diff
@@ -1,88 +1,101 @@
-import { buttonSubtextPreview, buttonDataComponent } from '../../locators/button';
+import {
+  buttonSubtextPreview,
+  buttonDataComponent,
+  buttonDataComponentIFrame, 
+  buttonSubtextPreviewIframe,
+} from '../../locators/button';
 import { icon } from '../../locators';
 import { positionOfElement } from '../helper';
 
 Then('Button label on preview is {word}', (label) => {
   buttonDataComponent().should('have.text', label);
 });
 
+Then('Button label on preview is {word} in IFrame', (label) => {
```

Comment:
Can we use here `label on preview is {word}` from `common-steps.js` step which is a reusable step in our code

Reply:
this step is looking for a `[data-element="label"]`, but we are looking for a `[data-component="button"]` in this step. So no, we can't